src/pages/Account/Components/CoinsTable.tsx

```diff
@@ -89,57 +96,116 @@ export type CoinDescriptionPlusAmount = {
 } & CoinDescription;
 
 export function CoinsTable({coins}: {coins: CoinDescriptionPlusAmount[]}) {
+  const [showVerifiedOnly, setShowVerifiedOnly] = React.useState(true);
+
+  const filteredCoins = showVerifiedOnly
+    ? coins.filter((coin) => coin.isInPanoraTokenList)
+    : coins;
+
+  const heavyTextColor = grey[450];
+  const lightTextColor = grey[400];
```

Comment:
These need to be more significant of a difference.

It's hard to tell which one's selected and which one isn't.

I think we should probably also add recognized as a selection (basically known + not banned).

Let me know if you want to go down this path or need help to do it